src/args.rs

```diff
@@ -78,17 +78,17 @@ pub(crate) fn clap_app(tmp_dir: &str) -> clap::App {
         .arg(
             Arg::with_name(FTPS_CERTS_FILE)
                 .long("ftps-certs-file")
-                .value_name("PKCS12_ARCHIVE")
-                .help("Sets the path the the certificates used for TLS security/FTPS")
+                .value_name("PEM_FILE")
+                .help("Sets the path the the certificates used for TLS security")
```

Comment:
pelease add.requires(FTPS_KEY_FILE) so clap can ensure that both parameters are present on startup.